api/lib/opentelemetry/metrics/metric.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Metrics
+    # Metric is a base class for various types of metrics. Metric subclasses are specialized for the type of a time series that Metric holds.
+    # Metric is constructed from the Meter class by providing set of Metric identifiers like name and set of label keys.
+    class Metric
+      # @api private
+      def initialize(label_keys_size:)
+        @label_keys_size = label_keys_size
+      end
+
+      def default_time_series
+        self.class.TIME_SERIES
```

Comment:
I think this should be `self.class::TIME_SERIES`.

But either way, can we avoid using constants like this? Just let the subclass override this method. Same for `time_series_for` below.